src/ChatCore.ts

```diff
@@ -64,4 +71,27 @@ export class ChatCore {
       notes: data.response.notes,
     };
   }
+
+  /**
+   * Make a request to Chat stream API to generate the next message.
```

Comment:
Maybe "Make a request to Chat streaming API to generate the next message and consume its tokens via server-sent events"? I feel like it's nice to indicate what exactly "stream" is doing vs "get"